src/main/resources/se/diabol/jenkins/pipeline/portlet/DeliveryPipelineViewPortlet/portlet.jelly

```diff
@@ -0,0 +1,19 @@
+
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:dp="/hudson/plugins/view/dashboard" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    <dp:decorate portlet="${it}">
+        <tr><td>
+            <div>
+                <st:include page="main.jelly" it="${it.deliveryPipelineView}" />
+                <style type="text/css">
+                    div.pipeline-main {
+                        border:none !important;
+                    }
+                    div.pagination {
+                        height: 1px !important;
+                        line-height: 1px !important;
+                    }
+                </style>
+            </div>
+        </td></tr>
+    </dp:decorate>
+</j:jelly>
```

Comment:
Actually I'm uncertain if it will inspect the .jelly file but it doesn't hurt adding a new line here.